_assets/js/ta-selectors.js

```diff
@@ -46,8 +47,13 @@ dropdownButton.addEventListener('click', () => {
 });
 
 window.onload = function () {
-  selectorState = getSearchParam('filters').split(';');
-  // If selector state is empty string, set to empty array
+  // If there is something in local storage, grab that state and use it:
+  if (accessStorage('filters') !== null && getSearchParam('filters') === '') {
+    selectorState = accessStorage('filters').split(';');
+  } else {
+    selectorState = getSearchParam('filters').split(';');
+  }
```

Comment:
So basically what this is trying to handle is when to use local storage vs. the url params, because we want both to be available.

This gets our saved filter state either by using localStorage or by using the URL search params. We want to use local storage in the case that someone is navigating by links to return to the resources page.

This won't invoke the browser's history (and then use the search params), so we need another way to grab the previous state of the checkboxes.

When navigating via link, the search params in the link will be empty, so if we have the state in local storage, let's use that.

I want it to be saying:

If something and something
x = values in local storage
else
x = values in the search params

Does that make sense?

Reply:
@billyhunt This has been fixed.